src/main/java/com/google/sps/servlets/NotificationServlet.java

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.servlets;
+ 
+import com.google.sps.classes.Notification;
+import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+ 
+/**
+* Servlet that handles the fetching and posting of notifications.
+*/
+@WebServlet("/notification")
+public class NotificationServlet extends HttpServlet {
+  /**
+   * Fetches the notifications of a specific user.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get user's ID from Utility method.
+    int userId = Utility.getUserId();
+    // Fetch notifications if user is signed in and convert the ArrayList to JSON
+    // using Utility method.
+    response.setContentType("application/json;");
+    response.getWriter().println(Utility.convertToJsonUsingGson(getNotifications(userId)));
+  }
+
+  /**
+   * Posts a new notification for all followers of a modified question or answer.
```

Comment:
Is this your only notification type?

Reply:
As of this PR, it is. Again, this is Andrés's file. I made another PR where I add a new type of notification and I'm pretty sure I modify this :)